### PR TITLE
Add analytic

### PR DESCRIPTION
**Pull request by MishkaV**

### CODE REVIEW

File: lib/models/analytic/aiuta_analytic_event_type.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  TryOnResultsError,
+  // Result
+  TryOnResultsShown,
+  FlowExit,
+  GeneratedImageShared,
+  ProductLike,
+  ProductAddToCart,
+  // Feedback
+  FeedbackPositive,
+  FeedbackNegative,
```
Comment (MishkaV) [inline lib/models/analytic/aiuta_analytic_event_type.dart:34]: params:
- feedback: String

File: lib/models/analytic/aiuta_analytic_event_type.dart [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  // Picker
+  ImagePickerPageEntered,
+  LoadingTryOnPageEntered,
+  // Result
+  ResultPageEntered,
+  // History
+  HistoryPageEntered,
+
+  // Actions
+  // Onboarding
```
Comment (MishkaV) [inline lib/models/analytic/aiuta_analytic_event_type.dart:17]: Should we add `FinishOnBoarding` ?

File: lib/models/analytic/aiuta_analytic_event_type.dart [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  WelcomeScreenStartButtonClicked,
+  ConsentGiven,
+  // Picker
+  TryOnStarted,
+  NewPhotoStarted,
```
Comment (MishkaV) [inline lib/models/analytic/aiuta_analytic_event_type.dart:22]: We have `NewPhotoTaken`/`GalleryPhotosSelected` events - can reuse them or create new

File: lib/models/analytic/aiuta_analytic_event_type.dart (hunk 2 above)
Comment (nGrey) [reply lib/models/analytic/aiuta_analytic_event_type.dart:17]: Let's add.

File: lib/models/analytic/aiuta_analytic_event_type.dart (hunk 1 above)
Comment (nGrey) [reply lib/models/analytic/aiuta_analytic_event_type.dart:34]: Let's add `option: Int` - to indicate index of selected option

File: lib/models/analytic/aiuta_analytic_event_type.dart [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  FlowExit,
+  GeneratedImageShared,
+  ProductLike,
+  ProductAddToCart,
+  // Feedback
+  FeedbackPositive,
+  FeedbackNegative,
+  // History
+  HistoryScreenClicked,
+  GalleryPhotoSelected,
```
Comment (nGrey) [inline lib/models/analytic/aiuta_analytic_event_type.dart:37]: I don't really get to point of it as
"Click on another photo in scan history to select it for try-on" we don't have this.

Let's remove it, and add OldPhotoSelected near the NewPhotoTaken

File: lib/models/analytic/aiuta_analytic_event_type.dart [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  GalleryPhotoDeleted
```
Comment (nGrey) [inline lib/models/analytic/aiuta_analytic_event_type.dart:38]: Maybe HistoryPhotoDeleted?

**Pull request merged**